pkg/discovery/monitoring/master/cluster_monitor.go

```diff
@@ -251,22 +250,24 @@ func (m *ClusterMonitor) genContainerMetrics(pod *api.Pod, podCPUMillicore, podM
 		containerMId := util.ContainerMetricId(podMId, container.Name)
 
 		//1. CPU, Memory, CPULimitQuota and MemoryLimitQuota capacity
+		cpuCapacityMillicore := podCPUMillicore
+		memCapacityKilobytes := podMem
 		limits := container.Resources.Limits
+		// Quantity.MilliValue() returns the value of ceil(quantity * 1000) as int64.
+		// As a result, the smallest possible cpuLimit in millicore is 1
 		cpuLimit := limits.Cpu().MilliValue()
```

Comment:
Is it that if no limit is configured, this will return 0?
If so, then what you have implemented is logically correct, but would it be easier to read to test against `!= 0`?

Reply:
Wouldn't `!=0` potentially let negative values slip in (if for whatever reason there is a possibility that `MilliValue()` can return negative value).

Reply:
Yes, if no limit is configured, this will return 0. Both > 0 and >= 1 are the same in this case.